Bmx.Idp.Okta/models/SessionOptions.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Bmx.Idp.Okta.models {
```

Comment:
Double-check: did you intentionally not capitalize `models`?

Reply:
Nope, just force of habit. Addressed this.